Generate.py

```diff
@@ -152,6 +156,10 @@ def main(args=None) -> Tuple[argparse.Namespace, int]:
     args.multi = max(player_id - 1, args.multi)
 
     if args.multi == 0:
+        if player_errors:
+            errors = '\n\n'.join(player_errors)
+            raise Exception(f"Encountered {len(player_errors)} error(s) in player files. "
```

Comment:
This should probably be a ValueError since everything up to this point is yaml parsing